Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -418,7 +418,7 @@ unsigned int compute2DCoordsMimicDistMat(
     RDKit::ROMol &mol, const DOUBLE_SMART_PTR *dmat, bool canonOrient,
     bool clearConfs, double weightDistMat, unsigned int nFlipsPerSample,
     unsigned int nSamples, int sampleSeed, bool permuteDeg4Nodes,
-    bool forceRDKit) {
+    bool /* forceRDKit */) {
```

Comment:
We normally suppress this warning with `RDUSED_PARAM(forceRDKit)`

Reply:
Ok, updating those.